### PR TITLE
Updated ESI documentation

### PR DESCRIPTION
**Pull request by tzumainn**
This change updates the documentation to emphasize recent changes
and near-term plans.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,89 @@
...
+- [Usage][usage]: common commands used by ESI administrators and users
 
-- The [requirements document][reqs] describes the basic requirements of the ESI project.
-- The [design document][design] describes how are are implementing those requirements.
-- The [upstream features][upstream] document describes changes we will need to make in upstream projects to support those requirements or to allow an upstream project to take advantage of ESI.
-- The [deployment document][deployment] describes a method of deploying and configuring ESI.
-- The [usage document][usage] lists common commands used by administrators and users.
+### Planning
+
+- [Requirementst][reqs]: initial requirements of the ESI project
```
Comment (DanNiESh) [inline README.md:51]: typo: [Requirements]

**Review by DanNiESh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tzumainn) [reply README.md]: Nice catch! Fixed.

File: docs/upstream-features.md [hunk 2]
```diff
@@ -2,37 +2,45 @@
...
 Some ESI features will require enhancement to upstream OpenStack projects.
 
-## Ironic: Multi-tenant support
+## Current Work
 
-ESI requires multi-tenant support in Ironic to meet our basic functionality. This enables a tenant to “own” a baremetal node — that is, to have exclusive access to it — without requiring administrative privileges.
+### Ironic: Attestation support with Keylime
 
-An initial implementation of this feature has merged as [If08586f3e9705dd38ff83e4b500d9ee3cd45bce3][]. There is a detailed description of the change available in the [associated blueprint][I1e898f7b9791aa579a733996711945e273ef6a4a].
+[Keylime][] provides attestation — that is, assurance that a given baremetal node has not been tampered with. We would like to integrate this into the Ironic provisioning workflow such that prior to provisioning a node Ironic will run it through an attestation step, and will fail the node if the attestation fails.
```
Comment (DanNiESh) [inline docs/upstream-features.md:9]: For ESI, do we only need to consider an attestation during node cleaning when it's returned from the lessee?

Comment (tzumainn) [reply docs/upstream-features.md:9]: I'd say the answer is up in the air :) I didn't actually change this text; just moved it around.

**Review by naved001 — APPROVED** [review]

**Review by larsks — APPROVED** [review]
Looks good to me!

**Review by DanNiESh — APPROVED** [review]
Looks good!

**Pull request merged**